helmc fails with apiVersion v1 and kind LimitRange

Prior to this command the namespace was already created.

```
 helmc install system                                                                                                                                                                     
---> Running `kubectl create -f` ...
error validating "STDIN": error validating data: API version "" isn't supported, only supports API versions ["componentconfig/v1alpha1" "extensions/v1beta1" "metrics/v1alpha1" "v1" "authorization.k8s.io/v1beta1" "autoscaling/v1" "batch/v1"]; if you choose to ignore these errors, turn validation off with --validate=false

[ERROR] Failed to upload manifests: exit status 1
```

kubectl works
```
kubectl create -f system-limits.yaml                                                                                                                                                      
limitrange "system-limits" created
```

Example:
```
apiVersion: v1
kind: LimitRange
metadata:
  name: system-limits
  namespace: system
spec:
  limits:
  - default:
      memory: 1000Mi
    defaultRequest:
      cpu: 250m
      memory: 500Mi
    type: Container
```

--- a/action/install.go
+++ b/action/install.go
@@ -1,11 +1,9 @@
 package action
 
 import (
-	"bytes"
 	"os"
 
 	"github.com/helm/helm-classic/chart"
-	"github.com/helm/helm-classic/codec"
 	"github.com/helm/helm-classic/dependency"
 	"github.com/helm/helm-classic/kubectl"
 	"github.com/helm/helm-classic/log"
@@ -118,7 +116,11 @@ func uploadManifests(c *chart.Chart, namespace string, client kubectl.Runner) er
 	for _, k := range c.UnknownKinds(InstallOrder) {
 		for _, o := range c.Kind[k] {
 			o.VersionedObject.AddAnnotations(map[string]string{chart.AnnFile: o.Source})
-			out, err := marshalAndCreate(o.VersionedObject, namespace, client)
+			data, err := o.VersionedObject.JSON()
+			if err != nil {
+				return err
+			}
+			out, err := client.Create(data, namespace)
 			log.Msg(string(out))
 			if err != nil {
 				return err
@@ -127,16 +129,6 @@ func uploadManifests(c *chart.Chart, namespace string, client kubectl.Runner) er
 	}
 
 	return nil
-}
-
-func marshalAndCreate(o interface{}, ns string, client kubectl.Runner) ([]byte, error) {
-	var b bytes.Buffer
-	if err := codec.JSON.Encode(&b).One(o); err != nil {
-		return nil, err
-	}
-	data := b.Bytes()
-	log.Debug("File: %s", string(data))
-	return client.Create(data, ns)
 }
 
 // Check by chart directory name whether a chart is fetched into the workspace.

--- a/action/remove_test.go
+++ b/action/remove_test.go
@@ -27,7 +27,7 @@ func TestTRemove(t *testing.T) {
 		{"kitchensink", mockNotFoundGetter, false, "All clear! You have successfully removed kitchensink from your workspace."},
 
 		// when manifests are installed
-		{"kitchensink", mockFoundGetter, false, "Found 10 installed manifests for kitchensink.  To remove a chart that has been installed the --force flag must be set."},
+		{"kitchensink", mockFoundGetter, false, "Found 11 installed manifests for kitchensink.  To remove a chart that has been installed the --force flag must be set."},
 
 		// when manifests are installed and force is set
 		{"kitchensink", mockNotFoundGetter, true, "All clear! You have successfully removed kitchensink from your workspace."},

--- a/testdata/charts/kitchensink/manifests/sink-limits.yaml
+++ b/testdata/charts/kitchensink/manifests/sink-limits.yaml
@@ -1,0 +1,13 @@
+apiVersion: v1
+kind: LimitRange
+metadata:
+  name: kitchensink-limits
+  namespace: kitchensink
+spec:
+  limits:
+  - default:
+      memory: 1000Mi
+    defaultRequest:
+      cpu: 250m
+      memory: 500Mi
+    type: Container

